source/_patterns/01-molecules/components/compact-form.yaml

```diff
@@ -17,13 +17,24 @@ schema:
       enum:
         - GET
         - POST
-    state:
+    variant:
       type: string
       enum:
         - valid
         - error
+        - info
     message:
-      type: string
+      type: object
+      properties:
+        text:
```

Comment:
I think we need two things: help text + message. (Message probably only populated when an error? Probably no need to enforce that here. Could have a dependency on `state`.)